tests/app/test_scheduled.py

```diff
@@ -61,6 +61,7 @@ def test_group_by_field():
     assert ScheduledApp.group_by_field == 'app_connection_id'
 
 
+@pytest.mark.skip(reason='No need to run this as new architecture is being developed.')
```

Comment:
let's delete it then?

Reply:
I dont want to delete them just yet, because code may be reused for new tests

Reply:
We have a version control system if we want to revisit older versions of a codebase

Reply:
deleted skipped tests